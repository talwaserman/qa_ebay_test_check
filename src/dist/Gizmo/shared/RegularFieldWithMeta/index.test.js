"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _index = _interopRequireDefault(require("./index"));

it('gizmo shared - RegularField with Metadata', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    label: 'Title',
    value: '101738591',
    isDifferentValue: false,
    metadata: {
      mandatoryOnProduct: true,
      multiValue: true,
      umsAlignedAspect: true,
      aspectTypes: ['REQUIREDHARD', 'REQUIRED'],
      umsValues: []
    }
  })).toJSON();

  expect(tree).toMatchSnapshot();
});