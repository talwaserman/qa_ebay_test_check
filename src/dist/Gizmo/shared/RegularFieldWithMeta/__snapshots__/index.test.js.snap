// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gizmo shared - RegularField with Metadata 1`] = `
<div
  className="regular-field-wrapper"
>
  <strong>
    <label
      className="aspect-title-wrapper"
    >
      <strong>
        <span
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          Title
        </span>
        <span
          className="required-icon"
        >
          *
        </span>
      </strong>
      <div
        className="importance-level"
      >
        [Required]
        <span>
          [M]
        </span>
      </div>
    </label>
  </strong>
  <div
    className="field-value"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    101738591
  </div>
</div>
`;
